packages/core/src/shared/datetime.ts

```diff
@@ -0,0 +1,10 @@
+/*!
```

Comment:
Future idea: Now that we have this module, maybe the datetime formatting functions should live in it https://github.com/aws/aws-toolkit-vscode/blob/f98622bef01190b209b5a91b89d66761d4dbd680/packages/core/src/shared/utilities/textUtilities.ts#L349-L374